src/app/features/tech-record/components/tech-record-summary-changes/tech-record-summary-changes.component.ts

```diff
@@ -98,6 +104,11 @@ export class TechRecordSummaryChangesComponent implements OnInit, OnDestroy {
         this.techRecordChanges = changes;
         if (this.vehicleType === VehicleTypes.PSV || this.vehicleType === VehicleTypes.HGV) {
           delete (this.techRecordChanges as Partial<TechRecordGETPSV | TechRecordGETHGV>).techRecord_numberOfWheelsDriven;
+        } if (
+          (this.vehicleType === VehicleTypes.CAR || this.vehicleType === VehicleTypes.LGV)
+          && (changes as TechRecordGETCar | TechRecordGETLGV).techRecord_vehicleSubclass) {
```

Comment:
Maybe cool for the future but potentially could use Union types for this type of checking but I know theres some constraints around them 😄

Reply:
Yeah I like this idea and I think we could do with looking at how we're using the types package and if we can make some union types that make sense, depends if there are sensible unions we can make that occur regularly